src/Controller/OccurrenceController.php

```diff
@@ -72,11 +72,12 @@ public function listAction(Request $request, $page = NUll) {
           $query_array['page'] = 1;
         }
       }
-      
+
       $result = $this->eventDatabase->getOccurrences($query_array);
       $occurrences = $result->getItems();
       $number_items = $result->get('totalItems');
-      pager_default_initialize($number_items, 20);
+      // @todo use dependency injection.
+      \Drupal\Core\Pager\PagerManagerInterface::createPager($number_items, 20);
```

Comment:
This will not work: You cannot call a function on an interface. Use dependency injection.